setup.cfg

```diff
@@ -1,3 +1,6 @@
+[aliases]
+test_fw=pytest
```

Comment:
In this case, we can't use only test because 'test' is replaced in 'cmdclass' for the alias defined by this word. So I think use 'test framework' or something like that.